clients/python/client/osparc/_http_client.py

```diff
@@ -0,0 +1,47 @@
+from typing import Optional
+
+import aiohttp
+import tenacity
+
+
+class HttpClient:
+    """Async http client context manager"""
+
+    def __init__(
+        self,
+        exc_req_typ: Optional[str] = None,
```

Comment:
`exce_req_typ`?  Please prefer long and clear names to short acronyms. Nowadays computers do not have the old constraints of the 8/90s!  Are you into maths? :-)))

Reply:
:rofl: fixed